src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,35 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              data-cy="Tab"
+              className={cn({
+                'is-active': tab.id === selectedTab.id,
+              })}
+              key={tab.id}
+            >
+              <a
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+                onClick={() => selectedTabId !== tab.id && onTabSelected(tab)}
```

Comment:
it's better to move handlers to their own functions

```
const handleTabClick = (tab) => () => {
 if(selectedTabId !== tab.id) { 
   onTabSelected(tab)
 }
}
```
```suggestion
                onClick={handleTabClick(tab)}
```